lldb/test/API/functionalities/data-formatter/swift/string-index/main.swift

```diff
@@ -0,0 +1,40 @@
+// main.swift
+//
+// This source file is part of the Swift.org open source project
```

Comment:
I don't think we need the license comment anymore.

Reply:
ok

Reply:
done